src/Components/OurClient/OurClients.js

```diff
@@ -22,66 +21,117 @@ const OurClients = () => {
     speed: 500,
     rows: 2,
     slidesPerRow: 1,
-    nextArrow: <ChevronRight className="arrowRight" />,
-    prevArrow: <ChevronLeft className="arrowLeft" />
+    nextArrow: <ChevronRight />,
+    prevArrow: <ChevronLeft />,
+    responsive: [
+      {
+        breakpoint: 1440,
+        settings: {
+          slidesToShow: 4,
+          slidesToScroll: 3,
+          infinite: true,
+          dots: !true
+        }
+      },
+      {
+        breakpoint: 1024,
+        settings: {
+          slidesToShow: 3,
+          slidesToScroll: 3,
+          infinite: true,
+          dots: !true
+        }
+      },
+      {
+        breakpoint: 768,
+        settings: {
+          slidesToShow: 2,
+          slidesToScroll: 2,
+          initialSlide: 2
+        }
+      },
+      {
+        breakpoint: 576,
+        settings: {
+          slidesToShow: 2,
+          slidesToScroll: 1
+        }
+      },
+      // {
+      //   breakpoint: 420,
+      //   settings: {
```

Comment:
@Ashutosh-Maurya-87 remove commented code or define any valid reason to keep this commented code